libs/platform/src/lib/components/approval-flow/approval-flow-node/approval-flow-node.component.html

```diff
@@ -1,45 +1,46 @@
 <div
     *ngIf="meta?.renderVerticalLineBefore"
-    class="approval-flow-node__vertical-line-before"
-    [class.approval-flow-node__vertical-line-before-solid]="_isVerticalLineBeforeSolid"></div>
-<div
-    *ngIf="meta?.renderVerticalLineAfter"
-    class="approval-flow-node__vertical-line-after"
-    [class.approval-flow-node__vertical-line-after-solid]="_isVerticalLineAfterSolid"></div>
+    class="fdp-approval-flow-node__vertical-line-before"
+    [class.fdp-approval-flow-node__vertical-line-before-solid]="_isVerticalLineBeforeSolid"></div>
 
 <div
-    *ngIf="_isFinal && meta?.nodeIndex === 0"
-    class="approval-flow-node__carousel-end-marker"></div>
+    *ngIf="meta?.renderVerticalLineAfter"
+    class="fdp-approval-flow-node__vertical-line-after"
+    [class.fdp-approval-flow-node__vertical-line-after-solid]="_isVerticalLineAfterSolid"></div>
 
 <ng-container *ngIf="!_blank && !_space">
-    <div *ngIf="renderArrow" class="approval-flow-node__arrow"></div>
+    <div *ngIf="renderArrow" class="fdp-approval-flow-node__arrow"></div>
 
     <div
         *ngIf="_isRoot && meta?.nodeIndex === 0"
-        class="approval-flow-node__carousel-start-marker"></div>
-
-    <div *ngIf="isEdit" class="approval-flow-node__edit-controls">
-        <fd-checkbox
-            [(ngModel)]="_isSelected"
-            [compact]="true"
-            [disabled]="_isEditActionsAvailable"
-            (ngModelChange)="_onCheck(_isSelected)"
-        ></fd-checkbox>
-
-        <button
-            *ngIf="!this.node?.disableActions"
-            class="approval-flow-node__menu-button"
-            fd-button
-            fdType="transparent"
-            glyph="overflow"
-            [compact]="true"
-            [fdMenuTrigger]="menu"
-            [disabled]="_isEditActionsAvailable"
-        ></button>
-    </div>
+        class="fdp-approval-flow-node__carousel-start-marker"></div>
+
+    <div
+        *ngIf="_isFinal && meta?.nodeIndex === 0"
+        class="fdp-approval-flow-node__carousel-end-marker"></div>
+
+    <ng-content></ng-content>
+</ng-container>
 
-    <div class="approval-flow-node__inner" (click)="_onClick()">
-        <div class="approval-flow-node__avatar">
+<ng-template #overflowMenuButton>
+    <!-- Actually just the fd-grid-item action button but with some hardcode to be used as menu trigger -->
+    <button
+        *ngIf="!node?.disableActions"
+        [ngClass]="_actionButtonClass"
+        class="fdp-approval-flow-node__menu-button"
+        fd-button
+        fdType="transparent"
+        glyph="overflow"
+        [compact]="true"
+        [fdMenuTrigger]="menu"
+        [disabled]="_isEditActionsAvailable"
+        (click)="$event.stopPropagation()"
+    ></button>
+</ng-template>
+
+<ng-template #nodeContent>
+    <div class="fdp-approval-flow-node__inner">
```

Comment:
I'm not sure about the `fdp-` prefix. This will trigger a lot of questions from the users side.

Reply:
Why not? 

We have `fd-*` prefix for all the stuff related to the Fundamental-NGX Core. 
And we're actually have the `fdp-*` prefix in a lot of Platform components.